src/cli/common.rs

```diff
@@ -432,8 +433,14 @@ pub fn list_installed_components(toolchain: &Toolchain<'_>) -> Result<()> {
     Ok(())
 }
 
-pub fn list_toolchains(cfg: &Cfg) -> Result<()> {
+pub fn list_toolchains(cfg: &Cfg, m: &ArgMatches<'_>) -> Result<()> {
```

Comment:
Rather than this, I'd prefer to see a wrapper function in `rustup_mode.rs` and this function take a `bool` for verbose mode.

(i.e. let's not leak our use of `clap` into this module file)